src/OpcUaPiXtendServer/CMakeLists.txt

```diff
@@ -70,6 +96,33 @@ if (WIN32)
     endif()
 endif()
 
+# -----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
+#
+# config moduls by env variables
+#
+# -----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
+if (DEFINED ENV{OPCUAPIXTENDSERVER_MODUL_V2S_DUMMY})
+    add_definitions(-DOPCUAPIXTENDSERVER_MODUL_V2S_DUMMY)
+    message(STATUS "Modul V2S activate dummy")
+endif ()
```

Comment:
Add additional message in else -> "Modul V2S activate real access"

also for the next messages